homeassistant/components/luftdaten/__init__.py

```diff
@@ -198,8 +198,9 @@ async def async_update(self):
         try:
             await self.client.get_data()
 
-            self.data[DATA_LUFTDATEN] = self.client.values
-            self.data[DATA_LUFTDATEN].update(self.client.meta)
+            if len(self.client.values) != 0:
```

Comment:
```py
if not self.client.values
```

Reply:
different flavor but ok for me.

Reply:
you mean this `if self.client.values:`, right?

Reply:
Yes, sorry.